Sources/MobileCoinClient+Config.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  Copyright (c) 2020-2021 MobileCoin. All rights reserved.
+//
+
+// swiftlint:disable function_parameter_count multiline_function_chains
```

Comment:
we going to fix these after?

Reply:
no, Im making some exceptions for specific files based on their content.